Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
-
+gem "activesupport"
```

Comment:
`activesupport` is part of rails. Do you really need it? If it's needed, I suggest moving the requirement to `.gemspec`.

Reply:
I need it for running the tests, I will move it to the .gemspec. thanks.